### ISSUE TITLE
creating index to file that doesn't exists does not output error

### ISSUE THREAD

**Issue by shannonekj** [COMBINE-lab/salmon#105]
I ran the following command: 

```
salmon index -t metagG.ffn -i transcript_index --type quasi -k 31
```
to a symlink to a file (metaG.ffn) that didn't actually exist and salmon was stuck on 
```
[Step 1 of 4] : counting k-mers
```
 for an hour instead of spitting out a "file does not exist" error.

**Comment by rob-p** [comment]
Hi @shannonekj,

  Thanks for reporting this.  This is resolved upstream & in the develop branch, and will be fixed in the next release of Salmon.  Sorry for the inconvenience!

--Rob

**Comment by shannonekj** [comment]
Awesome! Thanks @rob-p !

**Comment by rob-p** [comment]
This fix is in the newest release (v0.8.0).
